docs/accessibility-resources.md

```diff
@@ -1,17 +1,62 @@
 # Accessibility Resources 
 
-Designing for accessibility is an ongoing process integral to software design and development. The B.C. government aims to achieve WCAG 2.2 AA compliance for both internal and external websites and web content.
+The BC Government strives to reach [Web Content Accessibility Guidelines 2.2 AA](https://www.w3.org/TR/WCAG22/) (WCAG) for all digital products and services. WCAG sets the basis for web content. The Developer Exchange team designs for Accessibility using an ecosystem approach.
 
-We recommend the following resources to support and enhance your accessibility initiatives, including privacy information where available. When using web tools, remember to exercise caution to safeguard personally identifiable information (PII).
+Let’s consider multi-lingual website accessibility and screen readers. The mind map shows a user’s Work in Progress (WIP) as they design a website. They want it compatible with screen readers:
+<br>
+```mermaid
+%%{
+  init: {
+    'theme': 'base',
+    'themeVariables': {
+      'primaryColor': '#3467FE',
+      'primaryTextColor': '#fff',
+      'primaryBorderColor': '#7C0000',
+      'lineColor': '#000000',
+      'secondaryColor': '#002AFA',
+      'tertiaryColor': '#1C4D5A'
+    }
+  }
+}%%
+
+mindmap
+  root((Multi-lingual Accessiblity: <br> Screen readers))
+    Indigenous Languages
+            Unsupported by<br> screen readers
+            BC Sans Typeface
+            Techdocs: Indigenous<br> Languages in Systems
+              
+    Plain language
+        Formatting
+            Headings
+            Descriptive Links
+        Grade 8 <br> readability
+        Translation<br> Services
+
+    Development
+        Primary<br> language
+            Language<br> codes
+            Language<br> direction
+        Character<br> encoding
+              
+````
+
+BC Sans Typeface supports special characters and syllabics of Indigenous languages. The technical document Indigenous Languages in Systems shows how to support graphemes. They change written content, but screen readers don’t support Indigenous languages. Software bias creates a culture bias for marginalized communities. Now, Indigenous language only speakers who use screen readers can't access the website.
```

Comment:
Consider linking to the Indigenous Languages in Systems document mentioned here?

Reply:
Thanks for the catch!